### ISSUE TITLE
Unable to run a site/accessibility scan

### ISSUE THREAD

**Issue by anilnatha** [MAAP-Project/ZenHub#375]
I want to run a site/accessibility scan of our existing site (https://www.maap-project.org) using SortSite, but the software is unable to because of a nofollow rule in our robots.txt.

I tried to limit to the scan to exclude folders such as `/wp-content/*` and `/wp-includes/*` to resolve some of the scan issues but was unsuccessful.

The software recommended added the following line to run the scan. Could this be accommodated?

```
User-agent: PowerMapper
Allow: /
```

Here is a copy of the output when I attempted to run the scan:

[https://app.zenhub.com/files/272553335/a1c1441f-568c-419f-8c9a-daf5377a17f0/download](https://app.zenhub.com/files/272553335/a1c1441f-568c-419f-8c9a-daf5377a17f0/download)

In the meanwhile, I will keep testing different configurations to see if I can get the software to ignore the wordpress folders.

**Comment by anilnatha** [comment]
I was able to get around this issue and conduct a scan by disabling SortSites "Obey robots.txt" setting although I am still confused as to why the software continued to attempt to crawl paths that I had blocked.
